### PR TITLE
chore(performance): add initial k6 excel2json performance test (DEV-3885)

### PR DESCRIPTION
**Pull request by SamuelBoerlin**
Based on https://github.com/dasch-swiss/dsp-das/pull/1666

This PR adds a simple initial k6 performance test for excel2json using the existing [test data](https://github.com/dasch-swiss/dsp-tools/tree/main/testdata/excel2json/excel2json_files) in this repository.

This will be used as a basis for the planned automated performance tests and the resulting data will be uploaded to our [Grafana Cloud k6 project](https://dasch.grafana.net/a/k6-app/projects/3705130).

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]
Nice!  
One small thing: I wouldn't mark this PR as `feat`, because in the release notes this would show up like a new feature, which is not really the case. I'm not sure though, what to label it? chore? test?

**Review by seakayone — APPROVED** [review]

**Review by Nora-Olivia-Ammann — COMMENTED** [review]
Since I don't know javascript I cannot judge the code but the Readme is very good. Thanks!

File: k6/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# DSP-DAS Performance Testing
```
Comment (jnussbaum) [inline k6/README.md:1]:
```suggestion
# DSP-TOOLS Performance Testing
```

File: k6/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+This repository contains performance tests for the DSP-TOOLS project.
```
Comment (jnussbaum) [inline k6/README.md:3]:
```suggestion
This directory contains performance tests for DSP-TOOLS.
```

**Review by jnussbaum — APPROVED** [review]
I didn't try to run it on my machine, so I cannot tell if it really works. But generally, it looks good.

**Comment by jnussbaum** [comment]
> Nice! One small thing: I wouldn't mark this PR as `feat`, because in the release notes this would show up like a new feature, which is not really the case. I'm not sure though, what to label it? chore? test?

Right, "feat" is reserved for user-facing new features. I would be okay with "chore" or "test"

File: k6/README.md (hunk 1 above)
Comment (SamuelBoerlin) [reply k6/README.md]: Oops, thanks!

File: k6/README.md (hunk 2 above)
Comment (SamuelBoerlin) [reply k6/README.md]: 👍

**Comment by SamuelBoerlin** [comment]
Thanks for the feedback! I've changed the prefix to "chore".

**Pull request merged**